sweepbatcher/sweep_batcher.go

```diff
@@ -458,6 +465,11 @@ func (b *Batcher) spinUpBatchFromDB(ctx context.Context, batch *batch) error {
 		log:              batchPrefixLogger(fmt.Sprintf("%d", batch.id)),
 	}
 
+	cfg := batchConfig{
+		maxTimeoutDistance: batch.cfg.maxTimeoutDistance,
+		batchConfTarget:    defaultBatchConfTarget,
```

Comment:
Why setting `defaultBatchConfTarget` is needed here? If it is used, it means, that `confTarget`s of sweeps were not taken into account.

Reply:
The source of confTarget is `fetchSweep` method. I think, after resuming from DB we should make sure that confTarget is filled from `fetchSweep` before it is used to calculate fee rate (i.e. before `publish`). Then we can remove `const defaultBatchConfTarget`.

Reply:
We update this when the first sweep comes in, see [here](https://github.com/lightninglabs/loop/blob/master/sweepbatcher/sweep_batch.go#L408)

Reply:
Note that this is just a reorder, no added code.

Reply:
yeah, ideally let's not introduce unwanted changes